virtcontainers/fc.go

```diff
@@ -493,6 +492,24 @@ func (fc *firecracker) client() *client.Firecracker {
 	return fc.connection
 }
 
+func (fc *firecracker) createJailedResource(name string) (string, error) {
```

Comment:
I feel the name too generic,  what about createDrivePlaceholder ,  if not based on the name of `createJailedResource` should fail this function if fc.jailed is false?

Reply:
>should fail this function if fc.jailed is false?

[`fcJailResource`](https://github.com/kata-containers/runtime/blob/master/virtcontainers/fc.go#L494) doesn't fail if `! fc.jailed`, so I think this function shouldn't fail 

> I feel the name too generic,

since it can create or not a jailed resource, I think the name contains the word `jailed` . `createJailedDrive`  wdyt?

Reply:
sounds good to me